lib/main_common.pm

```diff
@@ -1426,6 +1426,9 @@ sub load_extra_tests_desktop {
         }
         if (!get_var("NOAUTOLOGIN")) {
             loadtest "x11/multi_users_dm";
+            if (check_var('DESKTOP', 'gnome')) {
+                loadtest "x11/keyboard_layout_gnome";
```

Comment:
"keyboard_layout_gnome" might be a bit misleading when we are not actually changing the layout within a gnome session, maybe "keyboard_layout_gdm"?

Reply:
renamed.